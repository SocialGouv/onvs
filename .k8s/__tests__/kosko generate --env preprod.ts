//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);
test("kosko generate --preprod", async () => {
  process.env.HARBOR_PROJECT = "onvs";
  expect(
    await getEnvManifests("preprod", "", {
      ...project("onvs").preprod,
      RANCHER_PROJECT_ID: "c-bd7z2:p-ngv88",
    })
  ).toMatchSnapshot();
});
