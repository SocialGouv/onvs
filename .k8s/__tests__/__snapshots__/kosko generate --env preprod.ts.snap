// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate --preprod 1`] = `
"---
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    field.cattle.io/creatorId: gitlab
    field.cattle.io/projectId: c-bd7z2:p-ngv88
    git/branch: v1.2.3
    git/remote: >-
      https://gitlab-ci-token:[MASKED]@gitlab.factory.social.gouv.fr/SocialGouv/onvs.git
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  labels:
    azure-pg-admin-user: onvs
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
  name: onvs-85-preprod-dev2
---
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  labels:
    app: app
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
  name: app
  namespace: onvs-85-preprod-dev2
spec:
  replicas: 1
  selector:
    matchLabels:
      app: app
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        app.gitlab.com/app: socialgouv-onvs
        app.gitlab.com/env: preprod-dev2
        app.gitlab.com/env.name: preprod-dev2
      labels:
        app: app
        application: v1-2-3-onvs
        owner: onvs
        team: onvs
        cert: wildcard
    spec:
      containers:
        - image: harbor.fabrique.social.gouv.fr/onvs/onvs-app:1.2.3
          livenessProbe:
            failureThreshold: 6
            httpGet:
              path: /healthz
              port: http
            initialDelaySeconds: 30
            periodSeconds: 5
            timeoutSeconds: 5
          name: app
          ports:
            - containerPort: 3000
              name: http
          readinessProbe:
            failureThreshold: 15
            httpGet:
              path: /healthz
              port: http
            initialDelaySeconds: 0
            periodSeconds: 5
            successThreshold: 1
            timeoutSeconds: 1
          resources:
            limits:
              cpu: 200m
              memory: 256Mi
            requests:
              cpu: 50m
              memory: 128Mi
          startupProbe:
            failureThreshold: 12
            httpGet:
              path: /healthz
              port: http
            periodSeconds: 5
          envFrom:
            - secretRef:
                name: azure-pg-user
            - secretRef:
                name: app-sealed-secret
            - configMapRef:
                name: app-configmap
      initContainers:
        - env:
            - name: WAIT_FOR_RETRIES
              value: '24'
          envFrom:
            - secretRef:
                name: azure-pg-user
          image: >-
            registry.gitlab.factory.social.gouv.fr/socialgouv/docker/wait-for-postgres:5.0.1
          imagePullPolicy: Always
          name: wait-for-postgres
          resources:
            limits:
              cpu: 20m
              memory: 32Mi
            requests:
              cpu: 5m
              memory: 16Mi
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: 'true'
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  name: app-sealed-secret
  namespace: onvs-85-preprod-dev2
  labels:
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
spec:
  encryptedData:
    MAIL_HOST: >-
      AgCmfO/0u0+1cNvvY+pam8GrklZY905zwc9fRWkYVV/NBJzhyzgpC9UuSoNjVl+h3zSqogxmBHi1weX1Xd/uwHX1TtwFjyJBGDglu9JmTE8ointnCLhahbdfL/St6wn5OkyXQ00nTfxUGg+7RI1uvesEPF7PbLz6KahbHDqPSQEqUoDoWtcsL3kf9oSqzmuBdpSiBGPivXbPPLoBCoKfBVzS799vS/v+Hdq1/KBQfW+bHrAgj4AbEZe5LMqZsDqDkCra7Br4wsKbm4u42fkEhyiU8oNUZita1+6Wy/CJRKJ58tiXfEhTs9gtdf++AcOFESYba9xIUIXqcCiuS3cHAaWw/87GDCvoU/oJL0/qyy6W0bFT+eI7GCHwbK6XUVEzOL4BPXVUAtuXoc0Hk5WQrrVVw+XnBX8jCUQWTjY1tINNTVyyHItq5PgNDy/7IoajketMohjraCHWJ+psUNh3g6PBBSFilNmFd9RwVInlSotS3La8sFTv+XSzCkWQFXTKSbn3ByAGCDdJvLrX8mHZNEGDEvjzdFaEGb3bD5SBlomci4z3J7c1/Pvd1uInxRC2wk3UltHGKRsp7nWcTbbeudChZQ5saxUXyD2S13ZrQTgu25g8/6baeE9Qpof5fvpGTaDCR4p6b3jR0EQzTIm/IFJC4XAOPM+GAdLbfjq3hTMvFVGwd59+MZCcJdoTN9cklQrcFdyLIEfkUgZgiC2dQXiZ
    MAIL_PORT: >-
      AgCBQppqHQCQy1TO8B/shjCcCx1+M4vxMBET18YY8AMtk+swYxx/Re+X7SEWWIx2MYobgedrwrmCtU22B5L89L/E1JpC4G6kpyo6WsXZMRZyeI1lxITMWP8Vk874V1xlXl+Z1LMFK7Glc7ONhge3bb8oXJ/QOHa4DVnVm1BZc00HYnJ6cblTY139e8QwRKiyVqVT+u+a6sPqcS9v8uSxGdFSMrj27nf7HwZrwuoNERJ3oMYPwicuBbn5RY6CsuqopAnlCBQnBthUu5hvSZVBVLATTQN4Ywt89+dMjs+3TVXHT9xeal1mkz7spyRVQFp2fFEazfqsOr3OqPuVj5XIpUkqRVSdZOsopLWjd38cjOM7p6I9+UvoH+ulQ7ln7qqOMfanFOPQnwoLpuZC7IZUWjdXJBlJdQhkfYos1C9CkVTCbqf+0FPGKk3YkBmINiYY6d6AMPgkKidTSA22exoEeJnCce/wc52yc8p/gr2S1KRdlq/J29xcLqZ5jB1QO4wsp2OUTncKR5eAeTUWFpSHLclsnPdzLghleZ+57JDRNBrsRv6P6IdMQDOss8dUzPvnUvm/1vcpYkYPh8aa2wyPkzcVF3+3xr+4NidJlq3LHHEsuHAbE9mpvmA9VLbTBwpXivUTSoChF60Jh43HS4IuS2AGC4hytzzGRHilDA/XpLOvZfO5OfuTT9utQWjrMDUGrtT8qNyl3w==
    MAIL_USERNAME: >-
      AgAnVPNQV6W3/hng42rhz0SdYFSvxgE8IclyeBICTSIbU9IgiI2YCMMwk/j0DwKoLgWx3nFPFCGfHv8tWfcSmfiaJq6fhEK+NCQbupiNiwttM+gQxQp7TYyaPSF81ZO2GlYPxdnFeih93njrcGvZEWdfyKo0V8hxPgx2ZYWjlfC8TCwHUZiwe/rDVqEvhFBpruZnNwwuoNiYmQXewZuuguFGxHM2SAJaABNcf2W7TNw60H0Vho7Vk0J9lWy/4hcHDuun+rrq6jkmJ9ktktBe/U3peDW6Ido0X4fCcnjohmkSRZh/o7nJu6UdI5J+pNlYprc2T+PkkZZEFo7+0GwRMzCYkWVFPmxX3EObtKWmkMeZcTRJAqLih4TKqanSr+IukQ/kJJwWnhtZmKrqUUDZ9MJwkvtZVeLkugPzDNb7McQQ2U/6bfMP39ThfeOSx2pwMX5qBGSqmQpUPuXeFm4MT450uMwB9bkqLSWVXQGYYliosbb/KGJajLbeTie7JbHz86/tbyGaDt4ycZ9icx/+IRaPdhvsa6CLbV8SybOogyApcqJ+o8AAtSCkx29fNs2UXuKSEK0HQe2XuYqCHSixL+l9GmjSteqcvPQk67DD4yjjTAizN9M2/4A7p6YxjRYn/+dUyBTn1p1Hf6WKt31auZaOwW4LxZVwUV2AeieATjMJ73h8yKTBmGtYq1hJcsXg/P1Hhr5GFIYK23H0dOGeSQ==
    MAIL_PASSWORD: >-
      AgBB/frhpbGeUuVWg9OoUY5veBMJR9W7lIgXKrLl3rGdFNsJa++M/AfJ7hyDCW14otKH6Chp5sNHxw6u+VvAplZC0HLTUXyt5Ax7E9lKUFAy+wF5MsOy8sutjkuIgeFwEm+o1E5RtcMNzFBJLegmYTaJyRJ9aPhPaVs8TdTFRlTEn0VBIF35HLbYgiNrdas1dLZyr6Xuw/HJYd0zQqiEZKOHmJ74R5+q+D4RBeknnwy0j9TnG5FPHknjuEsHg8zwUUGXGUoe/0Wk+0PW6tOxVfTMb9HFCpYgd9+BaFD8GI9/UfCRG9kdUtT7z6MLJGgL8SNzjgHuNLF/vtr3KmSf47qMh96qCDOC8rfjBWnNd91gEsJnWhvvrX6RjbWey62mnW+JYxaLgko32w89P2z3osRv4Oxv00C5YpMRAJdXkUWdnR/jLFlycOxtQ/f9q0m0+WnWaaC1V4F6jO02+b5wWUdXdEEd8leAkpN+/TweKMO9ABf/cQAjau+fiagWf5Rt3dLs8C+n3FadYxXLKMIZyVVuQOGSsRdo4Q6/klKhJmHesDygk4EJEADkOVPOAUesV0uOFmsULM11RC7wrA6IlcU5cUFxiOYBo9QOaEZTy+6SDEtQYttWFayMGaX4oE0YF8rDFhN84GqtMa9feoCVCP/ejd1Wnp/4DATLRezig/tZYxR96HDpdSBrFN0zQvA6FaT0AGdr1gA0WD2pF+zRoA==
    MAIL_TO: >-
      AgAWqFUFzCM03vvN5UMYyv7H275wfC+GwPMPzZWo2GhRxg6gdAhbH6UQKxFkkI+q6KdZEo3sCk4joOmvg9RRFsCdcsDOoFmn2kkQgw7c/mOwEBxeAbXvxEq1G+7/lIfMRqlA+3Pp/c7YZi5xrQJ41K60funXEnPk/1YW3GwQWRGSR31rt9OzoN/Dw3NbEyNI81r215+mTbHVddx0PrN24F7+ffa+7hfv3e7THaj8a3U1HQvQeWXJUVOo663dBeoBV1b/iBLPVXy7lHMlZKDEEXJY5X9Do7cWgwxKT5cHIuf8jd89JwT8eD6FdSnVbpnrNwRV3w92yN+pmvdyzxIUcj/XNZPOGAIaKpFHwdw5zH5XAjTr7pbhqieu2E7rixNDWrTkXmF6nvoqaMnzoM0YVb8O/NSLTmEEmfH8giWSzFGguJOLafw2FAwxKlAVXyYWZ69seS2TEtsyH40gJbs77oggT2y2nb06OEJd9BliRPPvJ3ws8a1iiH19bGHr0Oj+HusYlHYJFpRvauBCEKWbw+doWLMWm8tWT9lJpoc2sD2RQ7KbyiRn0n1CwnOfupZ00MdWfVjpyK/mBBvu2jB7P2WfGcjk2E6RanBek0Nm2bHH9fdOU5uHImF384DQj3ipXcDODdiJWUrWgCzmxzuWt8K+CnkdrVjOdnhA/ZrZD1RHZa67+4W49P2JzK4enFLaEVaJ5z9q44xBYBjWCpvZ4t2WJ7dV0Q==
  template:
    metadata:
      annotations:
        sealedsecrets.bitnami.com/cluster-wide: 'true'
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        app.gitlab.com/app: socialgouv-onvs
        app.gitlab.com/env: preprod-dev2
        app.gitlab.com/env.name: preprod-dev2
      name: app-sealed-secret
      labels:
        application: v1-2-3-onvs
        owner: onvs
        team: onvs
        cert: wildcard
    type: Opaque
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: app-configmap
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  labels:
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
  namespace: onvs-85-preprod-dev2
data:
  NODE_ENV: production
  PORT: '3000'
  NEXT_PUBLIC_SENTRY_TOKEN: toto
  NEXT_PUBLIC_SENTRY_DSN: https://e70207cd7a2c4ecdb99849c212a964bf@sentry.fabrique.social.gouv.fr/40
  NEXT_PUBLIC_MATOMO_URL: https://matomo.fabrique.social.gouv.fr/
  NEXT_PUBLIC_MATOMO_SITE_ID: '25'
  POSTGRES_SSL: 'true'
  MAIL_WEBHOOK_TOKEN: superSecret
  MAIL_FROM: ONVS <noreply@fabrique.social.gouv.fr>
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: app
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
  name: app
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  namespace: onvs-85-preprod-dev2
spec:
  ports:
    - name: http
      port: 80
      targetPort: 3000
  selector:
    app: app
  type: ClusterIP
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  labels:
    app: app
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
  name: app
  namespace: onvs-85-preprod-dev2
spec:
  rules:
    - host: preprod-onvs.dev2.fabrique.social.gouv.fr
      http:
        paths:
          - backend:
              serviceName: app
              servicePort: 80
            path: /
  tls:
    - hosts:
        - preprod-onvs.dev2.fabrique.social.gouv.fr
      secretName: wildcard-crt
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: 'true'
    app.gitlab.com/app: socialgouv-onvs
    app.gitlab.com/env: preprod-dev2
    app.gitlab.com/env.name: preprod-dev2
  name: azure-pg-user
  namespace: onvs-85-preprod-dev2
  labels:
    application: v1-2-3-onvs
    owner: onvs
    team: onvs
    cert: wildcard
spec:
  encryptedData:
    DATABASE_URL: >-
      AgCk/IGcsXOeIXfx78xNimSPTlC886bLX4YkscWILfjD2AB8af+1Z0aGklKGTbNFrQY42S5DICEZc+ockyBx3zhY2Qto3aa6yCX8qUPu40CLkO35t/1yxLS5T5UbIRuHdQHcge1o01fmOiv+Vuq5wrxLlk2tDMpaN5CY+w0tbuIUQn2mm1n7/nSQWzvkOCdGLdT/8iuQrBccFP/l/tIHaKBO4jfqIJEneuxe9vemejreizYBscKG6Sh8NN4iJ7SI/xOHxHSEXWGRdCCX+MKCA9mEylqZabeTmHB1ODSugCHqP12E6vkTcUFiPIMBC5cVqNz7O2UGx47qOTmJ3bQjEhBw6Uehju6GRWgU1zJp+oxBYSeCx6OniSKsQ8zSwbE0M4MVbxpjdRwAZRzTK7BDvb8snOiv4p+RV0d7nyH8tVrNPyN1nXo6DOF9WtrWo5chGhQZQfeEnNiF3bjR5ej9A1PVzIBdH5mEFxUTu4qVFksZy4z0IJiDddiQKenqzDu7MPEofB8UoiL09U2OUCKQWUAxUiYP+Dqn4Gp0J0lC8oF1wpDBGizJRp9/F4SQ05voP8UvVW2OWCS/pV8Jz2rQNXS1zZaAm6UfYx2AkA3hEklq4dMnoimsxkO3/dp7BANgC7BigLHVCnpk2Eo1VGJ+CN5wnWaqCOut0T2UOmP5xVGAIxEQ9ctiyuHU2m6gUvBZO/PYCnTofNc9BfNuBc3J5Ja6ohyMPDFWvQ+q8bSTOSSqp9rFWwpf1ImBkGZD207BKrk4YfXLqL2ql2y+sCGzsLdcQiUMSjuoyys/Kt1hE3y6Xtc4coxXKqh/H7ZmB0HPLSs9Dj0=
    PGDATABASE: >-
      AgAIhK2H/7lfcUQBn1U79rLtBm9GjcD27SEj/qmwgHEb5bVmI1C6gvBYlOlSjDxgf7Lxxj8bY1WHHVw4+uK87f8Zbplbc2JRzFBNJfajQdZwFykvDjZXJ30wpT6GBrPiGjuTgleQg8xe/C2vHbffpHM/siM/Te4TGbgfryoID2cJVRrQ+/6qVgv3A5Sgxwm50SUIhBzBmav1T53GigV1a8AxI5aZtKmmL06J+JSz3ayDgDXVf+UeIdoplHqPnUfbSdyXTP8UGlpMR1wotuF9rx+t960fqB1+sMIjIwJRoOjhaFSd2deFY9xHbbDUfRw1jdNteeBVFby/BpS94hbHrjcgtqWzDboujOBZEWYVjuDvo2QYQb7mSp2OzQpBGBuM35Bzf3Zxil+ZP6eRG7MYWg3idA5sBqX7A/B4AOjLcm458JG5kwp642isNMo7RggXFcUDBm4YWgT5yC2cSR7zB4kuIl4WVSwmQrowxlPsYu/bDtHvbegbdsbZtY56NPe8jghn2Wj5suDI404IT0F5D/E1ZE+e7P58wijMj/2BFDLxqRrjij05zL6N39MFNyPzP2Lge+aluVtq5hEcoou1Hhheb0GS4WWVZgjSTwtSstFSl1eGFZTOy501cl4HvCA64GeHmywJftk5sND2P1l1iyagdGfP87VN3wNEfncUjjCHdiQM2nIS9LTIzc3FCf8E5Qdt5Pw=
    PGHOST: >-
      AgAU8dSdcwrrtel37hTwUcQRClMBR978rSaQZgXNcvNgPeUaAJOE7f9g/ciiC/gcNSt0BGL99tPltTJE24AvZDm8+NtY09HfcokjDb8D85Z+5ERM0ze6upsdUjCn1+aSbu1+mLP0bJ2qo16CGEr749MeDWMHTe6vCPB67MfBhIGvlFeTHyta0s/6uOO38SYJYs/X9upz8n6Ga9s0Cj5xgcR0Rq0y7W/NizVVe6mndQEwxc2wD9efkK4H/qQMYNpOqOfGLSuTupGA5/804lDD3DQQ7V7D2v0cosrDyvUEM1H09+sX/vdAYY4Nl/hSo52Ziw5/4SteGaaeBOtt5dpkalJET57z+U6GbxLYkI3R1IRzpm9kTtfNezbaOGvlw4RXWQvqQ1Bz+z8qOgT4w2kONUn8PM0r58PTtUwSmnoPn1NJNPemgdlX/Ro2qw0SasS5K07pDEC707IbD3jyS+YpUnYMMue2oACVCwVyuJjoXC0armCmXdYzFXU353K77P9/Ols175rSBXeKGnMpjgNsXeH4dvzNCsiurJN5VMssMPj15V40nWqkC68Fqxoq7diT5UaG4SRYehtiJygMFoAfTyHDEvw606Ra4lDby4F0djsCK0IsxrfMdqv44SA/at6kyb7Hsj3YTS8Rq/sOcnNJX7nobl1DUaZP/NXygPJgplcF/X1sdZhkMwb77aUC/THjGD/3QryzdUW+l52AEaWiT3GRC8AN0mAXGLV0PGf/8Pk+WsnftVQON5vzxw==
    PGUSER: >-
      AgA58F6c3MsJ1L5PdqB7RADrDAMgE364H9W8LM1oNboWrgvoQncsCHSXTk1sQ7x/r5IDzKrfNdqseWrj/4KxX31OSSq6YVH0Px+0/PQmhvuFQAm2Z7lfrDkV5jVWwE2TknZ7H8/qhw1dYsTpeKbVYUNCwedXM65mqoyap+Kb8KtBoDo/TzcqV63Rf7ISLzcUZ3tZniUuxAXdZscDpxfF1Uq5EUKMvWb+7C2FtlU0Nmkgli92pRmRZdOtjNexf5QrpckXMTOQLCsBSrOLg/u6gCp38ArEYupMdGpj8I18ozdiy3TF21UH2OSL3YkdfHbCB4w98mxKxLzPi7mvX3QoOPyQe8Ddz++D9xu6ZvfiT/LWh8YqPFX5AaDeU3G/8H+iqRNXE5iLE9wypRFyvk1Qqf4Wlcch+zqGl3pSwgnZb05F9DGAMaqtx6djy+0z+86W66Qc/qVZ9cduDwFI0i2vrm90csjM4EqGO7ipWfydFoHqh9PDbyxKdrjtGJp0CeYXpYOKYgOvBN/G93l7oDuiR0bYRNWnU20PTBMWrKcYjr5SZkQzGtVdSybCrSf5KFIn57ouYZy7Y2QvZwXyaEZW1YDUwIrZZJU8IbmNNw7aqa/3AEEwEyshEAQjfoLMf3TAqu7VH/MdtGI95SZUaG2NNLPWIVrIjJTJDCUf0C1M68igJZe1sqyyUdL/dlUk1IOdMkA8HZqfnlX+BeBPeGJdaUAnYGP8XbXm0BtWdSq8PqIoBTW7/qNj+dUQpLIonuk=
    PGPASSWORD: >-
      AgDF4BelGAhzPMRs14qISn//fShYIvt0LSL80E1TM/FEqbszLOjgJqDrv2qfQPc5oPSDG0KymosJKOcgbbsl7d/oqPRaWAntyNxmXvJwHlaiPt+bnuoSFwJyiQYegnqmtgt3lwEo+l4X+orR2FknRbEMvbXjt7S/QEVv8Mxb5VdFcHfibHHONqBSfKStPMs28KfW2O3xVCzYU65P+ucfyajNPRVleCY61XoA2pTrH138L3hI/YnS8Rdn4fml+0V/lwpDjy+3YFEpxrvfgzysYfJWoC2DomsL5HdX+zyTx4VOTRhd+Mjuy2PJsUzJaWzDXQhZ6sqmlHargCHrR16tvKVEXGBMb7UZckp4VazbjO2mIUL58L79CC2fyC0hqDufwo1/a3mdwuKyheBYhl3lowHw2nNP9R9MYK5YmJW2wQ0Y6WvwCbCOtkqEO8XNKkczk7PF4W6PDPlV9sgv+bR4TEjgwxXMIMaew0jouMseEFziMEDKugJOmF/eJxPNlFcVfIGXWhRazA9QwNCmJ++RxF2x7Ct/cEJCtuSS0JSHejZsxa8ksR8SKkwquREm25oa+yLMowdE1HftFS0/okrHR0oETBPu+ABEg1a19UC1Cz/zjY9mpwYTilQdLHH+vQQFD7G8RErnVIWRZCXTdOEcsb1bRRRiq7fDYHokvZlTu5Qac7eGo0shKNmAu3MZ0uCizaqrlvuIA1H6wjTzFAH1Xr5afNemsA==
    PGSSLMODE: >-
      AgCmr3o8kYpkr1fJH2SadFkO/mtQwnYo0YWMT2P25toxmQZ2vNG0xa3Xc3UA+xqQsfYZZR+qYwO8eZEGA7uOa8RRS8ZwMx0lIwfDFQLKONFC8Ci0HEU1TBB6g0iVff/IsbXg/0By6SlCULLDXD++9ReKQkRUPXz/H8s1LRVz4jLJhIRieLFh9EM0QKq0WMqPk1Fn0bO/dBPqDEEG8X1IRaf5pO8xSAmpfA6uZn3U2fXL0um19GskmgiDht6PBZlIF2mqz3q9dIK99moYEekh8QW6uQjuEZmxDmgkay4XbOYVJv4opH1rmAKO/b76Zmjrt3GjAsPqxfzSqtgB+ZvAnjQAy79G+tvG3UxSUgyQ+glmQZ/RXp14kFAGO32UMS1elr+nWKkSifCyrte6BZcT0Ge98/F0BIu0/EQFIrYhg3NtVgWSIx8X317vfwZ6dU6gC1zVZbTh7q7fx8bsSAW2L65/CJmSLTSnM/1qtKM456+TfIoqW1zMiDvRUKUF6V6FBg68RwR67/+Q3fOdSy+2xPpy1pWB8gZyDdrj/BjTtffRsfgzHoZH8iqp++TuN7W8QqjUoHxxs2g2+qgQkmP1grdPliLg9MKvw7O5ob8ANOISgPh0RmFaQGMqSrFcHVmNG8qX6ihjaaPS2+nDWJejXFqRWydaP3IeI4mVVvkYWeCV9Py+Y8ti0Q6igBWM62NQAuao/HkxgMYQ
  template:
    metadata:
      annotations:
        sealedsecrets.bitnami.com/cluster-wide: 'true'
        app.gitlab.com/app: socialgouv-onvs
        app.gitlab.com/env: preprod-dev2
        app.gitlab.com/env.name: preprod-dev2
      name: azure-pg-user
      labels:
        application: v1-2-3-onvs
        owner: onvs
        team: onvs
        cert: wildcard
    type: Opaque
"
`;
